src/prefect/task_worker.py

```diff
@@ -339,6 +382,19 @@ def yell(message: str):
     """
     task_worker = TaskWorker(*tasks, limit=limit)
 
+    loop = asyncio.get_event_loop()
+
+    server = uvicorn.Server(
+        uvicorn.Config(
+            app=create_status_server(task_worker),
+            host="127.0.0.1",
+            port=4422,
+            access_log=False,
+            log_level="warning",
+        )
+    )
+    status_server = loop.create_task(server.serve())
```

Comment:
Let's add a parameter that controls whether or not this starts up and have it default to false for now since we're still experimenting.